app/code/Magento/InventoryCache/Plugin/InventoryApi/CleanCacheAfterStockSourceLinksSavePlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCache\Plugin\InventoryApi;
+
+use Magento\Framework\App\Cache\TypeListInterface as CacheTypeListInterface;
+use Magento\InventoryApi\Api\StockSourceLinksSaveInterface;
+
+/**
+ * Clean cache after stock source links saving
+ */
+class CleanCacheAfterStockSourceLinksSavePlugin
```

Comment:
Just a suggestion for an improvement. 

We use the "CleanCache" definition everywhere. That might be a little bit confusing since we don't clean the cache actually but invalidate. So, maybe using "InvalidateCache" is more suitable here